indico/modules/users/templates/users_admin.html

```diff
@@ -48,9 +52,13 @@
                     {% endif %}
                 </div>
             </div>
-            <div class="i-box-content">
+            <span class="i-box-content">
                 <div class="info-message-box">
-                    <div class="message-text">{% trans num_of_users=num_of_users %}There are {{ num_of_users }} users, {{ num_deleted_users }} of which are deleted{% endtrans %}</div>
+                    <div class="message-text">
+                        {%- trans num_of_users=num_of_users -%}
```

Comment:
`num_of_users=num_of_users` isn't needed; plain variables already work out of the box. you only need to do stuff here in cases where you are accessing e.g. some attribute (since you cannot do that inside a trans block)

Reply:
Thanks, this was already here before the PR, but I didn't even notice it.